src/Helpers/helpers.php

```diff
@@ -20,3 +20,15 @@ function voyager_asset($path, $secure = null)
         return route('voyager.assets', $path);
     }
 }
+
+if (!function_exists('get_file_name')) {
+    function get_file_name($name)
```

Comment:
Not sure I like this as a helper function... in fact, I think we should move existing helper functions out into service classes or something anyway.  We can do that in another PR